tests/transports-governance/test_governance_hierarchical_validation.py

```
@@ -0,0 +1,895 @@
+- Budget hierarchy validation (VK ≤ Team ≤ Customer)
+- Mutual exclusion constraints (VK team_id XOR customer_id)
+- Model/provider filtering validation
+- Field validation and data integrity
+- Database constraint enforcement
+- Relationship validation and consistency
+- Error handling and validation messages
+
+This file contains ~150+ test scenarios covering validation rules.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)